src/events/handlers.ts

```diff
@@ -0,0 +1,35 @@
+import { transactionsChannel } from './channels/transactionsChannel.js'
+import { logger } from '../modules/logger.js'
+import { processSignalTransaction } from '../services/processSignalTransaction.js'
+import { processTransactionToNotify } from '../services/processTransactionToNotify.js'
+import { prisma } from '../modules/prisma.js'
+
+export const spawnEventHandlers = () => {
+  transactionsChannel.on('newSignalMessage', (tx) => {
+    processSignalTransaction(tx).catch((e) =>
+      logger.error(e, 'Failed to process signal transaction')
+    )
+  })
+
+  transactionsChannel.on('newMessage', async (tx) => {
+    let devices = await prisma.device.findMany({
+      where: { admAddress: tx.recipientId }
+    })
+
+    devices = devices.filter((device) => device.admAddress === tx.recipientId)
```

Comment:
Do you really need to filter devices here if you already used `where: { admAddress: tx.recipientId }` on line 16.